fs/operations/multithread.go

```diff
@@ -178,10 +179,25 @@ func multiThreadCopy(ctx context.Context, f fs.Fs, remote string, src fs.Object,
 		})
 	}
 	err = g.Wait()
-	closeErr := mc.wc.Close()
 	if err != nil {
 		return nil, err
 	}
+
+	// Copy last withheld byte
```

Comment:
This Open will cause another HTTP transaction to the backend and hence quite a lot of extra work / delay for each file.

Ideally what we'd do is fetch the last byte above where we have the connection open, stash it away, and store it here.